web/widgets/src/commonMain/kotlin/Color.kt

```diff
@@ -1,18 +1,21 @@
 package org.jetbrains.compose.common.core.graphics
 
-public data class Color(val red: Int, val green: Int, val blue: Int) {
+public data class Color(val red: Int, val green: Int, val blue: Int, val alpha: Float) {
```

Comment:
Do you see any concerns if `val alpha: Float` would have a default value `1.0`?

Reply:
Seems good to me, plus that would get rid of the top-level function issue (for now, though eventually there would probably be some helper functions like `Color(value: Long)` that would reintroduce that issue)